client/src/index.js

```diff
@@ -1,7 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter as Router } from 'react-router-dom';
 
 import './index.css';
 import App from './App';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+ReactDOM.render( 
+    <Router>
+    < App / >
+    </Router> , document.getElementById('root'));
```

Comment:
It's preferred to format each argument on its own line, split up like you'd normally split it up. For example:
```javascript
ReactDOM.render( 
  <Router><App /></Router>, 
  document.getElementById('root')
);
```
or even better:
```javascript
ReactDOM.render( 
  <Router>
    <App />
  </Router>, 
  document.getElementById('root')
);
```